src/code/sched.c

```diff
@@ -438,45 +599,54 @@ void Sched_ThreadEntry(void* arg) {
             osSyncPrintf("%08d:待機中\n", (u32)OS_CYCLES_TO_USEC(osGetTime()));
         }
 
+        // Await interrupt messages, either from the OS, IrqMgr, or another thread
         osRecvMesg(&sc->interruptQueue, &msg, OS_MESG_BLOCK);
 
         switch ((s32)msg) {
-            case ENTRY_MSG:
+            case NOTIFY_MSG:
+                // Notification from another thread
                 if (sLogScheduler) {
                     osSyncPrintf("%08d:ENTRY_MSG\n", (u32)OS_CYCLES_TO_USEC(osGetTime()));
                 }
-                Sched_HandleEntry(sc);
+                Sched_HandleNotification(sc);
                 continue;
             case RSP_DONE_MSG:
+                // RSP task complete
                 if (sLogScheduler) {
                     osSyncPrintf("%08d:RSP_DONE_MSG\n", (u32)OS_CYCLES_TO_USEC(osGetTime()));
                 }
                 Sched_HandleRSPDone(sc);
                 continue;
             case RDP_DONE_MSG:
+                // RDP task complete
                 if (sLogScheduler) {
                     osSyncPrintf("%08d:RDP_DONE_MSG\n", (u32)OS_CYCLES_TO_USEC(osGetTime()));
                 }
                 Sched_HandleRDPDone(sc);
                 continue;
         }
         switch (((OSScMsg*)msg)->type) {
-            case 1:
+            case OS_SC_RETRACE_MSG:
+                // Vertical retrace
                 Sched_HandleRetrace(sc);
                 continue;
-            case 4:
+            case OS_SC_PRE_NMI_MSG:
+                // PreNMI
                 Sched_HandleReset(sc);
                 continue;
-            case 3:
-                Sched_HandleStart(sc);
+            case OS_SC_NMI_MSG:
+                // NMI
```

Comment:
The exact time is an implementation detail that I'd rather not use in a comment. I can change `NMI` to `Shutdown` or something but I'm doubt that it's really any better. I don't think it's that confusing if you know the meaning since you'd also know that it's not the "real" NMI, as there's no time left to do anything by then.

Reply:
I guess a comment like
```suggestion
                // Shutting down
```
would also be helpful for someone who doesn't know "NMI"

but that would also work on prenmi I guess so idk, do what you think is best